_tools/fetch-team-data.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @fileoverview Script to fetch team data from GitHub. Call using:
+ *     node fetch-team-data.js
```

Comment:
Based on the relative filepath of `"./_data/team.json"` below, it seems like this script is intended to have the project root as the CWD. However, these usage instructions seem to imply that the script should have `_tools` as the CWD so that the script would be found, which could potentially be confusing.

```suggestion
 *     node _tools/fetch-team-data.js
```